__tests__/nc_news.test.js

```diff
@@ -0,0 +1,38 @@
+const app = require("../app.js");
+const request = require("supertest");
+const db = require('../db/connection.js');
+const testData = require('../db/data/test-data/index.js');
+const seed = require('../db/seeds/seed.js');
+
+beforeEach(() => seed (testData));
+afterAll(() => db.end());
+
+describe("nc_news",()=>{
+    describe("GET /api/topics", ()=>{
+        test("Status:200 should return an array of topic objects having the properties slug and description", ()=>{
+            return request(app)
+            .get("/api/topics")
+            .expect(200)
+            .then(({body})=>{
+                const {topics}=body
+                expect(topics).toBeInstanceOf(Array)
+                expect(topics).toHaveLength(3)
+                topics.forEach((topic)=>{
+                    expect(topic).toMatchObject({
+                        slug: expect.any(String),
+                        description: expect.any(String)
+                    })
+                })
+            })
+        })
+        test("Status:404 when passing an invalid endpoint for api/topics", ()=>{
```

Comment:
So the 404 is totally valid, but this is not a part of your topics testing, as it is also the express default this test would have never failed.